adobecom/milo
# Project pep

## Description
This adds all the code required for the PEP Prompts functionality. Full description of the feature, solution architecture, design specs, etc. can be found [here](https://wiki.corp.adobe.com/display/WEBT/Project+PEP%3A+Solution+Architecture).

## How to test
On the test page, log in with a user that is entitled to Firefly. After getting signed in, ensure that the `milolibs` query parameter has been maintained; if not, manually add it to the URL. A dropdown should appear below the App Switcher, prompting you to close the message to avoid being redirected. If no action is taken, you will get redirected to the Firefly web application.

## Why is this high priority?
The committed due date for all parties involved is May 6th, thus this is time-dependent. Additionally, this is a high priority project from a business perspective, where the goal is to bring our users closer to their desktop apps.

## JIRA epic
Resolves: [MWPW-139246](https://jira.corp.adobe.com/browse/MWPW-139246)

## Test URLs
- Before: https://main--cc--adobecom.hlx.page/products/firefly
- After: https://main--cc--adobecom.hlx.page/products/firefly?milolibs=project-pep

- Before: https://main--cc--adobecom.hlx.live/drafts/ramuntea/firefly
- After: https://main--cc--adobecom.hlx.live/drafts/ramuntea/firefly?milolibs=project-pep

For PSI:
- Before: https://stage--milo--adobecom.aem.live/
- After: https://project-pep--milo--adobecom.aem.live/
